src/Microsoft.AspNetCore.SignalR.Common/Protocol/HandshakeResponseMessage.cs

```diff
@@ -9,23 +9,42 @@ namespace Microsoft.AspNetCore.SignalR.Protocol
     public class HandshakeResponseMessage : HubMessage
     {
         /// <summary>
+        /// Depreciated.
         /// An empty response message with no error.
         /// </summary>
-        public static readonly HandshakeResponseMessage Empty = new HandshakeResponseMessage(null);
+        public static readonly HandshakeResponseMessage Default = new HandshakeResponseMessage(error: null);
```

Comment:
Renaming this is an API breaking change?

Reply:
Yeah, we'll have to rename it back :'(

Reply:
😞